modules/internal/DefaultRectangular.chpl

```diff
@@ -1717,6 +1717,26 @@ module DefaultRectangular {
     rwLiteral("}");
   }
 
+  proc DefaultRectangularDom.dsiSerialWrite(f) throws
+  where _supportsSerializers(f) && f.serializerType != IO.defaultSerializer {
+    if chpl_warnUnstable then
+      compilerWarning("Serialization of rectangular domains with non-default Serializer is unstable, and may change in the future");
```

Comment:
I think moving this to an `@unstable` annotation, might remove some of the stack trace through the IO internals from the warning output. (I believe there is some special logic for `@unstable` to do that, but I could be wrong).

Reply:
I tried doing this initially here and in the other places, but the attribute wasn't causing any warnings with ``--warn-unstable``. Maybe something to do with being called from the standard library?

Reply:
Oh true, I guess they are suppressed in standard modules.